pkg/subctl/cmd/validate_cni.go

```diff
@@ -78,4 +94,126 @@ func validateCNIInCluster(clusterName string, submariner *v1alpha1.Submariner) {
 		submariner.Status.NetworkPlugin)
 	status.QueueSuccessMessage(message)
 	status.End(cli.Success)
+
+	validateCalicoIPPoolsIfCalicoCNI(config)
+}
+
+func findCalicoConfigMap(clientSet kubernetes.Interface) (*v1.ConfigMap, error) {
+	cmList, err := clientSet.CoreV1().ConfigMaps(metav1.NamespaceAll).List(metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	for _, cm := range cmList.Items {
+		if cm.Name == "calico-config" {
+			return &cm, nil
+		}
+	}
+	return nil, nil
+}
+
+func validateCalicoIPPoolsIfCalicoCNI(config *rest.Config) {
+	dynClient, clientSet, err := getClients(config)
+	if err != nil {
+		status.Start("Creating clientset for cluster")
+		status.QueueFailureMessage(fmt.Sprintf("Error creating clients for cluster %s", err))
+		status.End(cli.Failure)
+		return
+	}
+
+	calicoConfig, _ := findCalicoConfigMap(clientSet)
```

Comment:
I think we should report an error:

```suggestion
	calicoConfig, err := findCalicoConfigMap(clientSet)
	if err != nil {
	    status.Start(fmt.Sprintf("Error trying to detect the Calico CNI: %s", err))
	    status.End(cli.Failure)
	    return
	}
```